src/components/Cart/Cart.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import styles from "./Cart.module.css";
+
+import { connect } from "react-redux";
+
+import CartItem from "./CartItem/CartItem";
+import Navbar from "../Navbar/Navbar";
+
+const Cart = ({ cart }) => {
+  const [totalPrice, setTotalPrice] = useState(0);
+  const [totalItems, setTotalItems] = useState(0);
+
+  useEffect(() => {
+    let items = 0;
+    let price = 0;
+    //JSON.parse(localStorage.getItem("products")).cart.forEach((item)
+    cart.forEach((item) => {
```

Comment:
try rewrite using reduce and without creating 2 let vars, more elegant and readable